<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1394</width>
    <height>449</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>XP Path</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Actions</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QGroupBox" name="actionGroupBox">
          <property name="title">
           <string>Action Sequence</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout">
             <item>
              <widget class="QListView" name="actionListView">
               <property name="minimumSize">
                <size>
                 <width>300</width>
                 <height>0</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_5">
               <item>
                <widget class="QPushButton" name="insertButton">
                 <property name="text">
                  <string>Insert Action</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="removeButton">
                 <property name="text">
                  <string>Remove Selected</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QFormLayout" name="formLayout">
             <item row="0" column="0" colspan="2">
              <widget class="QPushButton" name="resetSimButton">
               <property name="text">
                <string>Reset</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0" colspan="2">
              <widget class="QPushButton" name="stepSimButton">
               <property name="text">
                <string>Step</string>
               </property>
              </widget>
             </item>
             <item row="2" column="0" colspan="2">
              <widget class="QPushButton" name="runToSelSimButton">
               <property name="text">
                <string>Run to Selected</string>
               </property>
              </widget>
             </item>
             <item row="3" column="0" colspan="2">
              <widget class="QPushButton" name="runSimButton">
               <property name="text">
                <string>Run</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QGroupBox" name="simStateGroupBox">
          <property name="title">
           <string>Simulation State</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QGroupBox" name="workingSetGroupBox">
             <property name="title">
              <string>Working Set</string>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <layout class="QFormLayout" name="simStateFormLayout">
                <item row="0" column="0">
                 <widget class="QLabel" name="minLevelReqLabel">
                  <property name="text">
                   <string>Minimum Level Req:</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="QLabel" name="minLevelReqOutputLabel">
                  <property name="minimumSize">
                   <size>
                    <width>50</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                 </widget>
                </item>
                <item row="1" column="0">
                 <widget class="QLabel" name="workingSetStatusLabel">
                  <property name="text">
                   <string>Working Set Status:</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="1">
                 <widget class="QLabel" name="workingSetStatusOutputLabel">
                  <property name="text">
                   <string/>
                  </property>
                 </widget>
                </item>
                <item row="3" column="0">
                 <widget class="QLabel" name="hearthstoneLabel">
                  <property name="text">
                   <string>Hearthstone:</string>
                  </property>
                 </widget>
                </item>
                <item row="3" column="1">
                 <widget class="QLabel" name="hearthstoneOutputLabel">
                  <property name="text">
                   <string/>
                  </property>
                 </widget>
                </item>
                <item row="4" column="0" colspan="2">
                 <widget class="QGroupBox" name="groupBox">
                  <property name="title">
                   <string>Errors</string>
                  </property>
                  <layout class="QFormLayout" name="formLayout_2">
                   <item row="0" column="0">
                    <layout class="QVBoxLayout" name="errorDisplayLayout"/>
                   </item>
                  </layout>
                 </widget>
                </item>
                <item row="2" column="0">
                 <widget class="QLabel" name="label">
                  <property name="text">
                   <string>Completed Set Status:</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="1">
                 <widget class="QLabel" name="completedSetStatusOutputLabel">
                  <property name="text">
                   <string/>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <widget class="QListView" name="workingSetTable">
                <property name="minimumSize">
                 <size>
                  <width>200</width>
                  <height>0</height>
                 </size>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="completedSetGroupBox">
             <property name="title">
              <string>Completed Set</string>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <item>
               <widget class="QListView" name="completedSetTable">
                <property name="minimumSize">
                 <size>
                  <width>200</width>
                  <height>0</height>
                 </size>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="2" column="0" colspan="2">
         <layout class="QHBoxLayout" name="horizontalLayout_9">
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="saveActionsButton">
            <property name="text">
             <string>Save Actions</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Vendor</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string>Don't Sell</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QTableView" name="dontSellTableView">
             <property name="minimumSize">
              <size>
               <width>650</width>
               <height>0</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_7">
             <item>
              <widget class="QPushButton" name="dontSellAddButton">
               <property name="text">
                <string>Add</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="dontSellEditButton">
               <property name="text">
                <string>Edit</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="dontSellRemoveButton">
               <property name="text">
                <string>Remove</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="dontSellSaveButton">
               <property name="text">
                <string>Save</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_3">
          <property name="title">
           <string>Buy</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QTableView" name="buyTableView">
             <property name="minimumSize">
              <size>
               <width>650</width>
               <height>0</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_8">
             <item>
              <widget class="QPushButton" name="buyAddButton">
               <property name="text">
                <string>Add</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="buyEditButton">
               <property name="text">
                <string>Edit</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="buyRemoveButton">
               <property name="text">
                <string>Remove</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="buySaveButton">
               <property name="text">
                <string>Save</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1394</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionActions"/>
    <addaction name="separator"/>
    <addaction name="actionBuy_List"/>
    <addaction name="separator"/>
    <addaction name="actionSell_List"/>
   </widget>
   <widget class="QMenu" name="menuAddon">
    <property name="title">
     <string>Addon</string>
    </property>
    <addaction name="actionGenerate_Addon"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuAddon"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionActions">
   <property name="text">
    <string>Load Actions</string>
   </property>
  </action>
  <action name="actionBuy_List">
   <property name="text">
    <string>Load Buy List</string>
   </property>
  </action>
  <action name="actionSell_List">
   <property name="text">
    <string>Load Sell List</string>
   </property>
  </action>
  <action name="actionGenerate_Addon">
   <property name="text">
    <string>Generate Addon</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
